handbook.md

```diff
@@ -742,8 +743,8 @@ The goal of Atlantian Youth Rapier is to recreate pre-17th century fencing.  All
         #. Fighters with a valid authorization from another kingdom who move into Atlantia may trade in for an Atlantian authorization card. See the Policies of the MoL for the procedure.
     #. The Kingdom Earl Marshal or Deputy Earl Marshal for Rapier Combat shall warrant a select group of marshals for newly approved authorization forms.  Future authorizations shall follow the standard authorization policies and procedures.
 #. Rapier Combat Weapons and Armor Standards
-    #. Atlantia follows the Corporate Rules for Rapier in the Society for Creative Anachronism, Incorporated (found at the Society Marshals Web Site at [(http://www.sca.org/officers/marshal/combat/rapier/index.html),](http://www.sca.org/officers/marshal/combat/rapier/index.html), with the following modifications:
-        #. No spears are allowed.
+    #. Atlantia follows the Corporate Rules for Rapier in the Society for Creative Anachronism, Incorporated (found at the Society Marshals Web Site at [(http://www.sca.org/officers/marshal/combat/rapier/index.html)](http://www.sca.org/officers/marshal/combat/rapier/index.html), with the following modifications:
+        #. Only rubber-headed spears are allowed.
         #. Fighters who choose to participate in Cut & Thrust combat acknowledge that they may be struck with a harder blow than they would normally receive in rapier or light rapier combat, and they should act with the interest of the safety of their opponent in mind.
         #. In addition to the Society requirements, for Cut & Thrust combat, the back of the head must also be covered by rigid material, lined by at least 1/4 inch (6mm) of closed cell foam or an equivalent padding.
```

Comment:
Society now defines this.

9.6.4. The interior of fencing masks must have a minimum of ¼” (6.4 mm) open-cell
foam or equivalent resilient padding to create separation between the mask and the
wearer. Modern fencing masks (e.g. FIE and USFA type masks) in good working
order meet this requirement without additional padding, but may require additional
padding as it ages.
9.6.5. For helms that utilize a suspension system which does not, on its own, prevent
contact between the wearer and any rigid part of the helm, additional padding must
be present. This padding must consist of at least ¼” (6.4 mm) open-cell foam or
equivalent resilient material.
9.6.6. Helms without a suspension system and which rely on foam must use a minimum
of ¼” (6.4 mm) closed-cell foam or equivalent resilient padding which provides
progressive resistance to create separation between the hard outer shell of the helm
and the wearer